n2v/models/n2v_config.py

```diff
@@ -223,6 +227,8 @@ def _is_int(v,low=None,high=None):
         )
         ok['n2v_manipulator']       = self.n2v_manipulator in ['normal_withoutCP', 'uniform_withCP', 'normal_additive',
                                                                'normal_fitted', 'identity']
+        # ok['structN2Vmask'] = self.structN2Vmask.ndim = self.ndim and all([x in [0,1] for x in self.structN2Vmask.flat])
+        ok['structN2Vmask'] = True
```

Comment:
This should test if the dimensions of the mask are odd and if all entries are either 0 or 1.